app/models/subject.rb

```diff
@@ -118,6 +118,14 @@ def light
     end
   end
 
+  def entities_count
+    e = 0
```

Comment:
All this method can be replaced with simply `comments.flat_map(&:entities).size`